app/src/components/governance/ProposalHeader.vue

```diff
@@ -0,0 +1,293 @@
+<template>
+  <header class="header">
+    <div class="header-top-column">
+      <div class="header-top-row">
+        <div>
+          <BackButton />
+        </div>
+
+        <div class="inner-status">
+          <Status :status="status" />
+        </div>
+
+        <div class="buttons">
+          <button
+            v-clipboard:copy="currentRoute"
+            v-clipboard:success="() => onCopy()"
+            class="share-button"
+          >
+            <i class="material-icons">link</i>
+          </button>
+          <div :class="{ active: copySuccess }" class="icon-container">
+            <i class="material-icons notranslate success">check</i>
+          </div>
+          <TmBtn
+            v-if="
+              status.badge === 'Deposit Period' &&
```

Comment:
can we avoid string comparisons like this? I guess status should be:

```
{
  title, // this we can actually edit
  value, // this is an enum we compare
  color
}
```

Reply:
OK, let me change it